docs/articles/peer-monitoring-sets.md

```diff
@@ -0,0 +1,152 @@
+* The seam monitoring set
+
+# Automatic Resource Alias Mapping
+In practice, most Transmission Providers have at least the following identifiers for each resource:
+
+* An identity for the resource in a modeling system and/or asset registration system.  Since many modeling tools are CIM-based, this is typically a master resource identifier (MrID) in the form of a UUID.  
+* An identity for the resource in an EMS system.  These may be the same as the modeling system identities, but since most EMS technologies pre-date CIM modeling practices, they typically do not.  Instead, they often follow some convention established by the specific EMS technology in combination with company-specific naming conventions and modeling practices.  
+
+As discussed above, typically none of these identifiers are the same across grid operators.  Also, each 
+RC could decide to use either as their "primary" value for `resource-id` in TROLIE operations.  
+
+Naively, each RC could track the primary `resource-id`s used by the other.  So, for example, 
+RC A could track RC B's EMS IDs in its database mapped to it own internal ones, and then ensure that
+those EMS IDs are explicitly used (and expected back) whenever it interacts with RC B's 
+TROLIE.  RC A would have to do something similar for all other neighbors.  
+
+This approach is functional and perfectly valid.  However, it presents some challenges:
+
+* If both RC A and RC B were to use this approach, then each would have to manually maintain copies of one another's IDs.  
+* RC A would have to either hard-code or manually configure each peer so that they know which set of IDs apply to that peer.  
+
+Using monitoring set reconciliation however, any unique identifier may be shared.  This could mean only 
+one of the RCs maintaining an alias.  Alternatively, RCs that are using ICCP for real-time 
+communications must already coordinate point names for sending the ratings.  Using the point name as one of 
+the aliases allows it to be reused to map resources.  Consider the following example contents for monitoring sets
+as returned by [getMonitoringSet](../spec#tag/Monitoring-Sets/operation/getMonitoringSet) for `B-in` and `B-out` 
+respectively.  Note these examples are oversimplified for readability, as they both only contain a single resource:
+
+**`B-in`**:
+
+```json
+{% include_relative examples/RC-B-monitoring-set.json %}
+```
+
+**`B-out`**
+
+```json
+{% include_relative examples/RC-A-monitoring-set.json %}
+```
+
+The IDs used by RC A and B are very different.  The MRIDs are of course different.  The EMS IDs also use different
+naming conventions.  Also, RC B has chosen to use CIM MRIDs as their `resource-id`, whereas RC A has chosen to 
+use EMS IDs as their `resource-id`.  
+
+However, when querying a new version of `B-in`, monitoring set reconciler function in the RC A TROLIE can see that 
+one of the aliases, specifically the value "DLR_OUT_LINE_5742_NORM" matches another alias it has in its 
```

Comment:
I debated this in my head a lot.  _Do_ we need to make sure they name types are coordinated?  I guess it's maybe cleaner, but it occurred to me with everyone's specific naming conventions that this is something else people would whine about and collisions are unlikely.  I can certainly be convinced otherwise.

Reply:
Updated